client/state/sharing/services/selectors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns an object of service objects or null if there are no Keyring services.
+ *
+ * @param  {Object} state Global state tree
+ * @return {?Object}      Keyring services, if known.
+ */
+export function getKeyringServices( state ) {
+	return state.sharing.services.items || null;
+}
+
+/**
+ * Returns true if a request is in progress to retrieve keyring services,
+ * or false otherwise.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       Whether a request is in progress
+ */
+export function isKeyringServicesFetching( state ) {
+	return state.sharing.services.isFetching || false;
```

Comment:
Because initial state of the reducer is already `false`, we shouldn't need a fallback.